jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/LoadCalculator.java

```diff
@@ -0,0 +1,13 @@
+package org.jbpm.services.task.assignment;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.kie.api.task.TaskContext;
+import org.kie.api.task.model.User;
+
+public interface LoadCalculator {
```

Comment:
would be good to add some java docs as it seems end users can implement this interface to provide custom calculation mechanism. Maybe we should move this to kie-internal somewhere next to https://github.com/kiegroup/droolsjbpm-knowledge/tree/master/kie-internal/src/main/java/org/kie/internal/task/api/assignment